ci-operator/config/openshift/external-dns-operator/openshift-external-dns-operator-release-1.1.yaml

```diff
@@ -0,0 +1,152 @@
+    with: pipeline:external-dns-operator
+  - pullspec: quay.io/openshift/origin-kube-rbac-proxy:latest
+    with: pipeline:kube-rbac-proxy
+releases:
+  initial:
+    integration:
+      name: "4.12"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.12"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: verify
+  commands: |
+    export GOLANGCI_LINT_CACHE=$(mktemp -d)
+    make verify
+  container:
+    from: src
+- as: unit
+  commands: |
+    export XDG_CACHE_HOME=$(mktemp -d)
+    make test
+  container:
+    from: src
+- as: e2e-aws-operator
+  steps:
+    cluster_profile: aws
+    dependencies:
+      OO_INDEX: ci-index-external-dns-operator-bundle
+    env:
+      OO_CHANNEL: stable-v1.1
+      OO_INSTALL_NAMESPACE: external-dns-operator
+      OO_PACKAGE: external-dns-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: test
+      cli: latest
+      commands: make test-e2e
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: optional-operators-ci-aws
+- as: e2e-gcp-operator
+  steps:
+    cluster_profile: gcp
+    dependencies:
+      OO_INDEX: ci-index-external-dns-operator-bundle
+    env:
+      OO_CHANNEL: stable-v1.1
+      OO_INSTALL_NAMESPACE: external-dns-operator
+      OO_PACKAGE: external-dns-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: test
+      cli: latest
+      commands: make test-e2e
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: optional-operators-ci-gcp
+- as: e2e-azure-operator
+  steps:
+    cluster_profile: azure4
+    dependencies:
+      OO_INDEX: ci-index-external-dns-operator-bundle
+    env:
+      OO_CHANNEL: stable-v1.1
+      OO_INSTALL_NAMESPACE: external-dns-operator
+      OO_PACKAGE: external-dns-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: test
+      cli: latest
+      commands: make test-e2e
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: optional-operators-ci-azure
+- as: e2e-infoblox-operator
+  steps:
+    cluster_profile: azure4
+    dependencies:
+      OO_INDEX: ci-index-external-dns-operator-bundle
+    env:
+      OO_CHANNEL: stable-v1.1
+      OO_INSTALL_NAMESPACE: external-dns-operator
+      OO_PACKAGE: external-dns-operator
+      OO_TARGET_NAMESPACES: '!install'
+    test:
+    - as: test
+      cli: latest
+      commands: |
+        export DNS_PROVIDER=INFOBLOX
+        export INFOBLOX_CONFIG_DIR=/var/run/infoblox-credentials
+        make test-e2e
+      credentials:
+      - mount_path: /var/run/infoblox-credentials
+        name: infoblox-credentials
+        namespace: test-credentials
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: optional-operators-ci-azure
+zz_generated_metadata:
+  branch: release-1.1
+  org: openshift
```

Comment:
org should not be openshift.

Reply:
I don't think we have a choice here. Since we use the Openshift CI for external-dns-operator (like a lot of other addon operators do). This is a metadata generated by `make ci-operator-config`, without it the PR wouldn't pass the validation.